sn_networking/src/record_store.rs

```diff
@@ -530,19 +538,31 @@ impl RecordStore for ClientRecordStore {
 /// 1536 =     35937.398370712
 /// 1792 =   4447723.077333529
 /// 2048 = 550463903.051128626 (about 13% of TOTAL_SUPPLY at moment of writing)
-fn calculate_cost_for_relevant_records(step: usize) -> u64 {
+///
+/// The above exponential pricing curve causes a huge variance of store_cost and total_earned,
```

Comment:
Hmm, lets clean this up to reflect the correct and current implementation, then?